tools/targets/ARM_MUSCA_A1.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+# Copyright (c) 2017-2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from os.path import abspath, basename, dirname, splitext
+from os.path import join as path_join
+import tempfile
+import re
+import shutil
+from argparse import Namespace
+from tools.psa.tfm.bin_utils.assemble import Assembly
+from tools.psa.tfm.bin_utils.imgtool import do_sign
+from tools.psa.tfm.bin_utils.imgtool_lib import version
+
+SCRIPT_DIR = dirname(abspath(__file__))
+MBED_OS_ROOT = abspath(path_join(SCRIPT_DIR, os.pardir, os.pardir))
+MUSCA_A1_BASE = path_join(MBED_OS_ROOT, 'targets', 'TARGET_ARM_SSG', 'TARGET_MUSCA_A1')
+
+def musca_tfm_bin(t_self, non_secure_bin, secure_bin):
+    tempdir = tempfile.mkdtemp()
```

Comment:
It would be more appropriate (and more in line with the rest of our targets) to use the BUILD directory instead of a temporary directory for merging of binaries (this is how it is done for managed bootloader).

Reply:
@orenc17 has this been answered?

Reply:
no